server/gokbg3/src/main/webapp/setup/DDC.csv

```diff
@@ -0,0 +1,1001 @@
+rdc;token;value_de;value_en
+DDC;0;Informatik, Informationswissenschaft & allgemeine Werke;Computer science, information & general works
+DDC;1;Wissen;Knowledge
+DDC;2;Das Buch;The book
+DDC;3;Systeme;Systems
+DDC;4;Informatik;Data processing & computer science
+DDC;5;Computerprogrammierung, Computerprogramme & Daten;Computer programming, programs & data
+DDC;6;Spezielle Computerverfahren;Special computer methods
+DDC;7;Unbesetzt;Unassigned
+DDC;8;Unbesetzt;Unassigned
+DDC;9;Unbesetzt;Unassigned
+DDC;10;Bibliografie;Bibliography
+DDC;11;Bibliografien & Kataloge;Bibliographies
+DDC;12;Bibliografien & Kataloge von Einzelpersonen;Bibliographies of individuals
+DDC;13;Unbesetzt;Unassigned
+DDC;14;Anonymer & pseudonymer Werke;Of anonymous & pseudonymous works
+DDC;15;Von Werken aus einzelnen Orten;Bibliographies of works from specific places
+DDC;16;Von Werken �ber einzelne Themen;Bibliographies of works on specific subjects
```

Comment:
Umlauts are not parsed correctly - is the charset correct?

Reply:
Yes